Detectors/TOF/simulation/src/Detector.cxx

```diff
@@ -1861,27 +1867,29 @@ void Detector::addAlignableVolumes() const
       volPath += vpL4;
       volPath += istr;
 
-      volPath = "";
-
       symName = snSM;
       symName += Form("%02d", isect);
       symName += snSTRIP;
       symName += Form("%02d", istr);
 
-      // AliDebug(2,"--------------------------------------------");
-      // AliDebug(2,Form("Alignable object %d", imod));
-      // AliDebug(2,Form("volPath=%s\n",volPath.Data()));
-      // AliDebug(2,Form("symName=%s\n",symName.Data()));
-      // AliDebug(2,"--------------------------------------------");
+      LOG(DEBUG) << "--------------------------------------------"
+                 << "\n";
+      LOG(DEBUG) << "Alignable object" << imod << "\n";
+      LOG(DEBUG) << "volPath=" << volPath << "\n";
+      LOG(DEBUG) << "symName=" << symName << "\n";
+      LOG(DEBUG) << "--------------------------------------------"
+                 << "\n";
 
-      printf("Check for alignable entry: %s\n", symName.Data());
+      LOG(INFO) << "Check for alignable entry: " << symName << "\n";
 
       if (!gGeoManager->SetAlignableEntry(symName.Data(), volPath.Data(), modUID))
-        printf("Alignable entry %s not set\n", symName.Data());
-      //        AliError(Form("Alignable entry %s not set",symName.Data()));
+        LOG(ERROR) << "Alignable entry " << symName << " NOT set\n";
+      LOG(INFO) << "Alignable entry " << symName << " set\n";
```

Comment:
these LOG(INFO) produce huge output, making impossible to detect errors, can it be changed to DEBUG?